static/js/directives/ui/new-ui-toggle/new-ui-toggle.component.html

```diff
@@ -0,0 +1,47 @@
+<span class="new-ui-toggle-element">
+  <div class="row display-flex">
+    <div class="col-md-5 right-padding-0 align-items-center">
+      <span>Current UI</span>
+    </div>
+    <div class="col-md-3 right-padding-0 align-items-center">
+      <i class="fa fa-toggle-on" ng-class="$ctrl.newUIIsActive ? 'active':'fa-rotate-180 inactive'" ng-click="$ctrl.handleToogleClick($event)" />
+    </div>
+    <div class="col-md-4 right-padding-0 align-items-center">
+      <span>New UI</span>&nbsp
+    </div>
+  </div>
+
+  <div class="co-dialog modal fade" id="newBetaUIModal">
+    <div class="modal-dialog">
+      <div class="modal-content padding-2percent">
+        <div class="modal-header modal-border">
+          <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+            <h4 class="modal-title center-align center-align">
+              <i class="info-icon fa fa-lg fa-info-circle beta-ui-info center-align"></i>
+              <br/>
+              <span class="center-align">
+                Try the new Beta Environment
+              </span>
+            </h4>
+        </div>
+
+        <div class="modal-body center-align" style="padding: 20px;">
+          Try our Beta Environment on ui.quay.io. The Beta Environment allows you to interact with new features in an
```

Comment:
@Sunandadadi Can you remove references to quay.io? Let's just call it the Beta Environment.